dist/tools/flake8/check.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2017 Alexandre Abadie <alexandre.abadie@inria.fr>
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+if tput colors &> /dev/null && [ $(tput colors) -ge 8 ]; then
```

Comment:
To answer https://github.com/RIOT-OS/RIOT/pull/8078/files#r151843512: The `&>` here is a bash-ism. The portable way would be `> /dev/null 2> /dev/null`.